src/ray/protobuf/usage.proto

```diff
@@ -60,6 +60,12 @@ enum TagKey {
   SERVE_REST_API_VERSION = 13;
   // The number of serve apps running in the cluster as a string.
   SERVE_NUM_APPS = 14;
+  // Whether num_replicas changed as a lightweight config update
+  SERVE_NUM_REPLICAS_UPDATED = 15;
+  // Whether user_config changed as a lightweight config update
+  SERVE_USER_CONFIG_UPDATED = 16;
+  // Whether autoscaling_config changed as a lightweight config update
+  SERVE_AUTOSCALING_CONFIG_UPDATED = 17;
```

Comment:
```suggestion
  // Whether num_replicas changed as a lightweight config update
  SERVE_NUM_REPLICAS_LIGHTWEIGHT_UPDATED = 15;
  // Whether user_config changed as a lightweight config update
  SERVE_USER_CONFIG_LIGHTWEIGHT_UPDATED = 16;
  // Whether autoscaling_config changed as a lightweight config update
  SERVE_AUTOSCALING_CONFIG_LIGHTWEIGHT_UPDATED = 17;
```

Reply:
I believe we need approval from either @pcmoritz or @thomasdesr.